new-docs/spelling/Makefile

```diff
@@ -7,4 +7,7 @@ spelling-requirements: .make.spelling-requirements.installed
 	pip install -r requirements-spelling.txt
 	touch $@
 
-.PHONY: spelling spelling-requirements
+sort-allowed-words:
```

Comment:
Can we make this a dependency of `spelling-requirements`?

Reply:
c11a808